installation/sql/postgresql/base.sql

```diff
@@ -111,7 +111,12 @@ INSERT INTO "#__assets" ("id", "parent_id", "lft", "rgt", "level", "name", "titl
 (85, 18, 120, 121, 2, 'com_modules.module.108', 'Privacy Status', '{}'),
 (86, 18, 122, 123, 2, 'com_modules.module.96', 'Popular Articles', '{}'),
 (87, 18, 124, 125, 2, 'com_modules.module.97', 'Recently Added Articles', '{}'),
-(88, 18, 126, 127, 2, 'com_modules.module.98', 'Logged-in Users', '{}');
+(88, 18, 126, 127, 2, 'com_modules.module.98', 'Logged-in Users', '{}'),
+(89, 18, 128, 137, 2, 'com_modules.workflow.1', 'COM_WORKFLOW_BASIC_WORKFLOW_MODULES', '{}'),
+(90, 89, 129, 130, 3, 'com_modules.state.1', 'COM_WORKFLOW_BASIC_STAGE', '{}'),
+(91, 89, 131, 132, 3, 'com_modules.transition.1', 'Publish', '{}'),
+(92, 89, 133, 134, 3, 'com_modules.transition.2', 'Unpublish', '{}'),
+(93, 89, 135, 136, 3, 'com_modules.transition.3', 'Trash', '{}');
 
 SELECT setval('#__assets_id_seq', 89, false);
```

Comment:
```suggestion
SELECT setval('#__assets_id_seq', 94, false);
```
Auto-increment columns work differently in PostgreSQL than they do in MySQL. They use something called sequence, and here you have to adjust the next value of that sequence to your changes.

The same has to be done elsewhere in the PostgreSQL base.sql file where you have added new records to the inserts.